pkg/apis/machine/v1alpha1/types.go

```diff
@@ -1108,10 +1108,11 @@ type GCPMachineClassSpec struct {
 
 // GCPDisk describes disks for GCP.
 type GCPDisk struct {
-	AutoDelete bool              `json:"autoDelete"`
+	AutoDelete *bool             `json:"autoDelete"`
```

Comment:
Need to make this change on - `pkg/apis/machine/types.go` as well right? And then run regenerate clients.

Reply:
Okay, I've addressed this. PTAL again @prashanth26

Reply:
Thanks.